### DISCUSSION TITLE
Vietnamese Translation

### DISCUSSION THREAD

**Discussion by nguyenvanduocit** [vuejs-translations/guidelines#13] [General]
For Vietnamese translations, please discuss here.

And we are preparing to translate the doc into Vietnamese at [vuejs-vn](https://github.com/orgs/vuejs-vn).

## Old Translation

The translation for v2 can be found at: [vi.vuejs.org](https://vi.vuejs.org/v2/guide/) and [it's repo](https://github.com/orgs/vuejs-vn).

**Comment by ikhoai** [comment]
Cậu ơi đã bắt đầu dịch tí nào Vue 3 chưa?

**Reply by nguyenvanduocit** [reply]
Cũng dịch kha khá rồi ấy bạn.

**Reply by ikhoai** [reply]
Có cần giúp 1 tay, tớ cũng đang mún dịch tranh thủ học lun :D

**Reply by nguyenvanduocit** [reply]
Yesss, cần đấy bạn, bạn vào dịch với mọi người đi.

**Comment by nguyenvanduocit** [comment]
Các bạn xem hướng dẫn dịch ở đây nhé:
https://github.com/vuejs-vn/docs/discussions/11

**Comment by TrungThachDau** [comment]
Phải để là Vuejs-vi chứ vì vn là kí tự viết tắt của Vietnam còn vi mới là kí tự viết tắt của Vietnamese.

**Comment by Nekitori17** [comment]
Cái này còn sống không vậy?

**Reply by nguyenvanduocit** [reply]
Không còn bạn.

**Comment by tamnd** [comment]
Hiện nay bản dịch tiếng Việt đã khá cũ, không còn được cập nhật, thay đổi lần cuối 4 năm trước, chậm 1000+ commits so với bản gốc tiếng Anh: https://github.com/vuejsvn/docs

Các bản dịch khác đều được đặt dưới org https://github.com/vuejs-translations và được cập nhật khá thường xuyên, ví dụ:
- Tiếng Trung https://github.com/vuejs-translations/docs-zh-cn (4 ngày trước)
- Tiếng Đức https://github.com/vuejs-translations/docs-de (10 giờ trước)

Mình rất muốn khởi động lại phần dịch tiếng Việt cho Vuejs 3, thành tài liệu học tập cho cộng đồng sử dụng Vuejs.
Cách thức tiếp cận sẽ chia làm 2 phases:
- Phase 1 (gần hoàn thành) - sử dụng LLM + review nhanh để có được khung nội dung đầu tiên - hiện tại đặt ở repo cá nhân của mình https://github.com/tamnd/vuejs-docs-vi
- Phase 2 - review kĩ bằng con người (mình sẽ review từng trang một trước), trước khi gửi lên để review lại chuẩn thuật ngữ và văn phong.

Cách thức mình tiếp cận sẽ là dùng LLM để tăng tốc ban đầu, nhưng **KHÔNG** gửi bất kì nội dung nào chưa được (human) review trước khi có  review từ người khác.

cc @nguyenvanduocit mình có khởi động lại dự án dịch, nếu bạn còn hứng thú rất mong tham gia cùng!
cc @phanan (core team vue): em có tóm tắt lại các bước thực hiện để hướng tới bản dịch Vuejs cập nhật cho tiếng Việt tốt nhất.

**Comment by phanan** [comment]
I'll send you an invite.

**Comment by phanan** [comment]
@tamnd I've just sent you an invite to vuejs-translations/docs-vi. I'd prefer everything happen in this repo from the get-go (instead of having a personal repo).

**Reply by tamnd** [reply]
Thanks for the quick reply. I received the invitation. Which working style do you prefer?

1. Commit all machine translations at once (from my current private repository), or
2. Start with a blank project and add each page as small, reviewable pull requests?

I prefer the second approach. It may be slower, but it allows better quality control. My personal repository would only be used for reference.

**Comment by phanan** [comment]
2.

**Reply by tamnd** [reply]
Roger.
